src/main/python/editor/key_override.py

```diff
@@ -117,14 +117,28 @@ class LayersUI(QWidget):
     def __init__(self):
         super().__init__()
         container = QGridLayout()
+        buttons = QHBoxLayout()
         self.layer_chks = [CheckBoxNoPadding(str(x)) for x in range(16)]
         for w in self.layer_chks:
             w.stateChanged.connect(self.on_change)
+        btn_all_layers = QToolButton()
+        btn_all_layers.setText(tr("KeyOverride", "Enable all"))
+        btn_all_layers.setToolButtonStyle(Qt.ToolButtonTextOnly)
+        btn_no_layers = QToolButton()
+        btn_no_layers.setText(tr("KeyOverride", "Enable none"))
```

Comment:
just a tiny nit: how about "disable all" instead of "enable none"?

Reply:
Works for me! I used “Enable” for both to match “Enable on layers”, but “Disable all” is probably a more familiar term for users.